presto-parquet/src/test/java/com/facebook/presto/parquet/TestTupleDomainParquetPredicate.java

```diff
@@ -81,13 +84,13 @@
     public void testBoolean()
             throws ParquetCorruptionException
     {
-        String column = "BooleanColumn";
-        assertEquals(getDomain(BOOLEAN, 0, null, ID, column, true), Domain.all(BOOLEAN));
+        ColumnDescriptor columnDescriptor = createColumnDescriptor(PrimitiveTypeName.BOOLEAN, "BooleanColumn");
+        assertEquals(getDomain(columnDescriptor, BOOLEAN, 0, null, ID), Domain.all(BOOLEAN));
 
-        assertEquals(getDomain(BOOLEAN, 10, booleanColumnStats(true, true), ID, column, true), singleValue(BOOLEAN, true));
-        assertEquals(getDomain(BOOLEAN, 10, booleanColumnStats(false, false), ID, column, true), singleValue(BOOLEAN, false));
+        assertEquals(getDomain(columnDescriptor, BOOLEAN, 10, booleanColumnStats(true, true), ID), singleValue(BOOLEAN, true));
```

Comment:
Are there any existing tests the test the code path `TupleDomainParquetPredicate.getDomain(..)` that pass multiple ranges to it?